### ISSUE TITLE
newKey is not defined

### ISSUE THREAD

**Issue by benderTheCrime** [juanfran/gulp-scss-lint#63]
Hey, I'm having an issue with `^0.3.8`.

Here is the entirety of the gulp output:
```
[12:50:18] Starting 'scss-lint'...
[12:50:19] 'scss-lint' errored after 283 ms
[12:50:19] ReferenceError in plugin 'gulp-scss-lint'
Message:
    newKey is not defined
```

I'm using ruby-v2.2.0 and scss_lint (0.43.2)

I've got a config, but my only option is declaring a config file, which is equivalent to the default .yml for the gem.

**Comment by juanfran** [comment]
thank you @benderTheCrime 

I think it's already fixed, could you try the 0.3.9 version? thanks!

**Comment by benderTheCrime** [comment]
Indeed it is, thanks! Closing this.
